docs/ABOUT.md

```diff
@@ -0,0 +1,46 @@
+[Dart](https://en.wikipedia.org/wiki/Dart_(programming_language)) is a general-purpose computer programming language that is:
+ 1. [Concurrent](https://en.wikipedia.org/wiki/Concurrent_computing)
+ 2. [Class-Based](https://en.wikipedia.org/wiki/Class-based_programming)
+ 3. [Object-Oriented](https://en.wikipedia.org/wiki/Object-oriented_programming)
+
+Dart applications are typically compiled to [bytecode](https://en.wikipedia.org/wiki/Bytecode) that can run on any [Dart virtual machine](https://en.wikipedia.org/wiki/Dart_virtual_machine) (JVM) regardless of computer architecture.
```

Comment:
I think that this basic idea of being able to run on a multitude of different computer architectures is interesting and worth keeping. The bytecode bit is a bit nitty-gritty, though. At the end of the intro, we should link to something that lets people dig deeper if they're interested.

Let's drop the wikipedia link here, too.